### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by illayarajah**

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -29,7 +48,19 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:51]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -39,7 +70,16 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:65]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +119,24 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:104]: Reuse insertAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +147,25 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:115]: Reuse insertAt

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 1 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:51]: thanks, done.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 3 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:104]: Done, thank you

**Pull request closed without merging**